examples/lib/stories/collision_detection/circles.dart

```diff
@@ -7,11 +7,24 @@ import 'package:flame/geometry.dart';
 import 'package:flame/input.dart';
 import 'package:flutter/material.dart' hide Image, Draggable;
 
-const circlesInfo = '''
-This example will create a circle every time you tap on the screen. It will have
-the initial velocity towards the center of the screen and if it touches another
-circle both of them will change color.
-''';
+class Circles extends FlameGame with HasCollidables, TapDetector {
```

Comment:
Rename into `CirclesExample`? The name `Circles` sounds like it's a class that somehow manages multiple circle shapes.

Reply:
Fixed!